app/src/main/java/org/mozilla/fenix/home/collections/CollectionMenu.kt

```diff
@@ -0,0 +1,142 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.home.collections
+
+import android.content.Context
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.DropdownMenu
+import androidx.compose.material.DropdownMenuItem
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.runtime.Immutable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import mozilla.components.browser.state.selector.normalTabs
+import mozilla.components.feature.tab.collections.TabCollection
+import org.mozilla.fenix.R.string
+import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.home.sessioncontrol.CollectionInteractor
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Menu shown for a [TabCollection].
+ *
+ * @see [DropdownMenu]
+ *
+ * @param showMenu Whether this is currently open and visible to the user.
+ * @param menuItems List of options shown.
+ * @param onDismissRequest Called when the user chooses a menu option or requests to dismiss the menu.
+ */
+@Composable
+fun CollectionMenu(
+    showMenu: Boolean,
+    menuItems: List<CollectionMenuItemData>,
+    onDismissRequest: () -> Unit
+) {
+    DisposableEffect(LocalConfiguration.current.orientation) {
+        onDispose { onDismissRequest() }
+    }
+
+    // DropdownMenu uses the medium shape from MaterialTheme.
+    // Override it's corner radius to be the same 8.dp as in mozac_browser_menu_corner_radius
+    MaterialTheme(shapes = MaterialTheme.shapes.copy(medium = RoundedCornerShape(8.dp))) {
```

Comment:
Should we still be using FireFoxTheme here?

Reply:
`CollectionMenu` is expected to be in a composition with `FirefoxTheme` applied because it is hardcoded to read `FirefoxTheme.colors`.
`DropDownMenu` is expected to be in a composition with `MaterialTheme` applied because it is implemented as a `Card` which [is hardcoded to use `MaterialTheme.shapes.medium`](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/material/material/src/commonMain/kotlin/androidx/compose/material/Card.kt;l=61;drc=4f2a25b8f8ee02053c57b524a1a6341e1edb9c06) as it's shape for which here we override the actual property.

While not pretty we will have to do something like this in multiple places in our app where we'd use android composables but with our own styling on top and so needing to override the defaults.